### PR TITLE
docs: updated docs for very_good test regarding #570

### PR DESCRIPTION
**Pull request by dipakp2726**
## Description

updated docs for very_good test

## Type of Change

- [ ] ✨ New feature (non-breaking change which adds functionality)
- [ ] 🛠️ Bug fix (non-breaking change which fixes an issue)
- [ ] ❌ Breaking change (fix or feature that would cause existing functionality to change)
- [ ] 🧹 Code refactor
- [ ] ✅ Build configuration change
- [x] 📝 Documentation
- [ ] 🗑️ Chore

## Related issue
 #570

### CODE REVIEW

**Comment by dipakp2726** [comment]
Hi felix @felangel 🙋‍♂️

File: site/docs/commands/test.md [hunk 1]
```diff
@@ -32,6 +32,8 @@ Unlike `flutter test`, `very_good test` will always run your tests without insta
 
 This is an optimization done by the CLI because dependency installation is usually run once after cloning the repository. Conversely, running tests locally is usually done many times and it's often unnecessary to re-install dependencies prior to each test run.
 
+To use Optimized tests it is mandatory that none of your tests main method is async.
```
Comment (felangel) [inline site/docs/commands/test.md:35]: This is addressed by #599 and should not be an issue going forward.

**Review by felangel — CHANGES_REQUESTED** [review]
Closing due to #599

File: site/docs/commands/test.md (hunk 1 above)
Comment (dipakp2726) [reply site/docs/commands/test.md:35]: thank you!

**Pull request closed without merging**